include/Macro.h

```diff
@@ -79,6 +79,11 @@
 #define EOS_USER        5
 
 
+// nuclear eos table default mode
+#define NUC_TABLE_MODE_TEMP 0
+#define NUC_TABLE_MODE_ENGY 1
+
```

Comment:
I think these two definitions are redundant since `NUC_TABLE_MODE_` is used as a compilation option.

Reply:
Then, where should I define these compiling options?

Reply:
I think you do not need to define these two, but maybe I am wrong. I think it could be better to put these two at `NuclearEoS.h`.

Reply:
I add these here because other compiling options are here. Maybe Hsi-Yu gives better suggestion for the place.